arch/arm/src/sama5/sam_adc.c

```diff
@@ -1335,7 +1417,7 @@ static int sam_adc_settimer(struct sam_adc_s *priv, uint32_t frequency,
   priv->tc = sam_tc_allocate(channel, mode);
   if (!priv->tc)
     {
-      aerr("ERROR: Failed to allocate channel %d mode %08" PRIx32 "\n",
+      aerr("ERROR: Failed to allocate channel %d mode %08x\n",
             channel, mode);
```

Comment:
Why this is changed? the original version seems to be correct?

Reply:
Don't know...I guess it threw an error at some point and I changed it, but it seems OK now. I was having quite a bit of grief with version control....

Reply:
Have changed it back